geometry_msgs/PoseStamped posestamped
bool lateral
